target/linux/ipq806x/image/Makefile

```diff
@@ -316,4 +316,21 @@ define Device/zyxel_nbg6817
 endef
 TARGET_DEVICES += zyxel_nbg6817
 
+define Device/askey_rt4230w
+	$(call Device/LegacyImage)
+	$(call Device/UbiFit)
+        DEVICE_VENDOR := Askey
+        DEVICE_MODEL := RAC2V1K/RT4230W
```

Comment:
Which is it? Decide for one name (either consistently use board name or branded name).
If board name is not unique for the setup, it will be branded name.

Reply:
Decided on RT4230W-REV6 as the device name.